sympy/functions/elementary/exponential.py

```diff
@@ -768,7 +768,7 @@ def _eval_expand_log(self, deep=True, **hints):
                 else:
                     return unpolarify(e) * a
         elif isinstance(arg, Product):
-            if arg.function.is_positive:
+            if force or arg.function.is_positive:
```

Comment:
Can you check for some examples  where `arg.function.is_positive` is `None`. Because Sympy has tri classification i.e. `True, False and None`.
Might be there cases when `args.function.is_positive = None` and it may enter the if and might give the wrong answer.
If you could find some cases then more precisely we can write
```
arg.function.is_positive is True:
```
I had not delved deeper, let me know if you could find such cases or may be I am missing something.